include/libtorrent/alert_types.hpp

```diff
@@ -2541,11 +2547,29 @@ namespace libtorrent
 		aux::allocation_slot m_v6_nodes_idx;
 	};
 
+	// The session_stats_header alert is posted during the init of the
+	// session thread.
+	//
+	// the ``message()`` method returns a string representation of the
```

Comment:
a bit of a nit-pick. in C++ terminology it's called "member function" no method :P